generate/templates/libraries/httpclient/ApiClient.mustache

```diff
@@ -252,7 +252,7 @@ namespace {{packageName}}.Client
         /// <summary>
         /// Adds a dictionary of custom headers to current list of custom headers.
         /// </summary>
-        internal void AddCustomHeaders(Dictionary<string, string> headersToAdd)
+        internal void AddCustomHeaders(IDictionary<string, string> headersToAdd)
```

Comment:
I think there's a custom headers example as well, that should probably be updated

Reply:
I've updated the example